### ISSUE TITLE
OneMore fails to load

### ISSUE THREAD

**Issue by dkter** [stevencohn/OneMore#119]
**Describe the bug**
After I install OneMore it doesn't load. The "COM Add-ins" dialog box provides the error message:
> Not loaded. A runtime error occurred during the loading of the COM Add-in.

![the above error](https://user-images.githubusercontent.com/34610081/101996196-2e0c7600-3c9e-11eb-8a37-6efda2e53b27.png)

**To Reproduce**
1. Install OneMore
2. Start OneNote

**Expected behavior**
OneMore should be loaded

**Environment (if applicable):**
 - OneNote Version: Office 365 64-bit
 - OneMore Version: 3.9.4

**Additional context**
It doesn't seem like a log file was generated.

**Comment by stevencohn** [comment]
I have to ask the obvious question first: Did you use the OneMore 64-bit installer since you have the 64-bit version of Office?
What happens if you try to tick the checkbox next to OneMoreAddin and click OK in the dialog you show?

**Comment by dkter** [comment]
>Did you use the OneMore 64-bit installer since you have the 64-bit version of Office?

Yes, but I just noticed when I run the installer it tries to install OneMore in the `Program Files (x86)` folder, even though the file I downloaded says `x64`:

![image](https://user-images.githubusercontent.com/34610081/102000855-b2b8bd80-3cb9-11eb-8837-d60009afb63a.png)

>What happens if you try to tick the checkbox next to OneMoreAddin and click OK in the dialog you show?

Nothing happens, and if I restart OneNote the checkbox becomes unchecked again (though it doesn't show the error anymore).

**Comment by 1001QAdotNET** [comment]
I can confirm there is problems with the new release
After installing the new release the addon button disappeared and I can't put it back ....

**Comment by stevencohn** [comment]
Apologies. I update the x64 installer. Please try it again.
https://github.com/stevencohn/OneMore/releases/tag/3.10

**Comment by 1001QAdotNET** [comment]
I think you linked the same 3.10 version that has the bug
I guess you wanted to point us to 3.11?! 
3.10 also reset the quick access toolbar (removes all the onenote related buttons) :-(

**Comment by stevencohn** [comment]
I just removed and updated the 3.10 version. Did you try it?

I'm not sure about the quick access toolbar. I'm guessing that if the commands are not available, OneNote tries to be too smart and kindly removes them for you? Nothing I can do about that. I have no control over that toolbar.

**Comment by dkter** [comment]
It installs in the `Program Files` folder now but still doesn't seem to work :/

**Comment by stevencohn** [comment]
I've been having trouble trying to find a 64-bit installation of OneNote so I can't prove that the latest release solves the problem but I did make a change to the OneMore installer that might help. So I could use your help in testing if you're willing

https://github.com/stevencohn/OneMore/releases/tag/3.10.1

**Comment by dkter** [comment]
nope, still doesn't work. Is there a way I could provide any sort of logging or debugging info that could help?

**Comment by stevencohn** [comment]
RRRRGH! Sorry, I wish Microsoft didn't "second class" OneNote. It doesn't install along with Office and the 64-bit installer doesn't actually install 64-bit OneNote so even their installer is broken it seems :-(

There really isn't any viable logging in OneNote any more. There used to be in v2013 but alas...

If I figure out how to install 64-bit OneNote, I'll be able to figure it out I'm sure but until then I can only hope someone is willing to help diagnose this out of the kindness of their heart

**Comment by stevencohn** [comment]
Did it work for you in the past, or are you a new OneMore user? 

Just so I know whether I actually broke something or it was broken all along.

**Comment by dkter** [comment]
nope, I'm a new user. Tried installing it for the first time and it didn't work

I should probably note that the first time I installed OneMore I accidentally had left OneNote open and when it didn't work I assumed that was the issue, but when I closed OneNote and tried uninstalling and reinstalling OneMore I had the same issue.

**Comment by stevencohn** [comment]
OK thanks. Not exactly the kind of "new user experience" I would hope for either

**Comment by stevencohn** [comment]
Alright, whew! I feel better for me, but still bad for you.

I was able to install 64-bit OneNote 2016 on a new machine and used the latest 3.10.1 installer and it seems to work just fine. So there's something strange with your configuration.

My only advice is to ensure that these Registry keys are deleted (do not exist) before you install OneMore:

HKEY_CLASSES_ROOT\AppID\{88AB88AB-CDFB-4C68-9C3A-F10B75A5BC61}
HKEY_CLASSES_ROOT\WOW6432Node\CLSID\{88AB88AB-CDFB-4C68-9C3A-F10B75A5BC61}
HKEY_CLASSES_ROOT\River.OneMoreAddIn
HKEY_CLASSES_ROOT\River.OneMoreAddIn.1
HKEY_CURRENT_USER\SOFTWARE\Classes\AppID\{88AB88AB-CDFB-4C68-9C3A-F10B75A5BC61}
HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\OneNote\AddIns\River.OneMoreAddIn
HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\River.OneMoreAddIn.dll

then try the installer again

**Comment by dkter** [comment]
Oh hey I got it! That wasn't the issue though, all those registry keys don't exist after I run the uninstaller

The issue was that I needed to select "everyone" instead of "just me" (the default) in the installer. If I do that it installs fine. I'm guessing my install of OneNote must have been installed "for everyone" back when I set it up ages ago.

Thanks for the help and quick responses!

**Comment by stevencohn** [comment]
Good news! I'll make a note of it in the project Wiki.

**Comment by 1001QAdotNET** [comment]
I am still having issues with this release

It shows up in addins as enabled but is it not showing on the ribbon bar
Enabling and disabling the addon is not solving the issue
Installing for everyone did not solve the issue

I am on Win10x64, OneNote 2013 x86

18120:1] Starting dllhost 18120, en-US/en-US, v3.10.1, OneNote 15.0, Office 15.0
18120:1] ..OnConnection(ConnectionMode:ext_cm_Startup,1)
18120:1] ..OneAddInsUpdate(custom[]:0)
18120:3] ..OnStartupComplete(custom[]:0)
18120:3] ..OneNote backup folder:: C:\Users\1001QA\AppData\Local\Microsoft\OneNote\15.0\Backup
18120:3] ..OneNote default folder: C:\Users\1001QA\Documents\OneNote Notebooks
18120:3] ..OneNote unfiled folder: C:\Users\1001QA\Documents\OneNote Notebooks\My Notebook\Quick Notes.one
18120:3] OnBeginShutdown(0)
18120:3] ..shutting down UI
18120:3] ..OnDisconnection(RemoveMode:ext_dm_HostShutdown,0)
18120:3] ..closing log

27516:1] Starting dllhost 27516, en-US/en-US, v3.10.1, OneNote 15.0, Office 15.0
27516:1] ..OnConnection(ConnectionMode:ext_cm_Startup,1)
27516:1] ..OneAddInsUpdate(custom[]:0)
27516:3] ..OnStartupComplete(custom[]:0)
27516:3] ..OneNote backup folder:: C:\Users\1001QA\AppData\Local\Microsoft\OneNote\15.0\Backup
27516:3] ..OneNote default folder: C:\Users\1001QA\Documents\OneNote Notebooks
27516:3] ..OneNote unfiled folder: C:\Users\1001QA\Documents\OneNote Notebooks\My Notebook\Quick Notes.one

**Comment by stevencohn** [comment]
Now you're really baffling me. It's registered and starting but not showing up! I think it's time to get a beer.

**Comment by stevencohn** [comment]
Give 3.10.2 a shot https://github.com/stevencohn/OneMore/releases/tag/3.10.2

Fingers crossed. I can't get it to break but .2 removes some unnecessary additions I made to .1

**Comment by CLITrees** [comment]
Same thing
it installs but not showing up on the ribbon
visible in the COM add-ons list

**Comment by stevencohn** [comment]
with 3.10.1 or 3.10.2?

**Comment by 1001QAdotNET** [comment]
3.10.2

**Comment by stevencohn** [comment]
OK. I'm trying to add some more error checking/logging. Thanks for your patience. This is one of those can't-repo "works-on-my-machine!" problems.

**Comment by 1001QAdotNET** [comment]
ok np

**Comment by stevencohn** [comment]
So here's the deal....
1. There's a new release 3.10.3 that adds some exception logging to the code that builds the ribbon bar group. I'm hoping your missing ribbon problems will be exposed in the log
2. I've done a bunch of testing and proven that on a 64-bit machine, both x86 and x64 OneMore installers work for either 32-bit or 64-bit OneNote. (I don't have a 32-bit machine to test but I'm guessing no one has 32-bit machines anymore so I'm not going to support that.)

**Comment by 1001QAdotNET** [comment]
The machine (OS) is x64 
OneNote is x32 (x86)

Still does the same
Maybe it is worth saying that when the first release came out I was able to install it and it was visible on the ribbon
Not sure what happened but it suddenly disappeared. There might have been a crash or something I do not remember the incident exactly but something happened and after the restart of onenote it was gone
Let me try to restart the machine to see if that changes the things
3.9.3 installs just fine so not sure what is going on

25448:1] Log restarted
25448:1] OnBeginShutdown(0)
25448:1] ..shutting down UI
25448:1] ..OnDisconnection(RemoveMode:ext_dm_HostShutdown,0)
25448:1] ..disposing 3 streams
25448:1] ..closing log

11936:1] Starting dllhost 11936, en-US/en-US, v3.10.3, OneNote 15.0, Office 15.0
11936:3] ..OnConnection(ConnectionMode:ext_cm_Startup,1)
11936:3] ..OneAddInsUpdate(custom[]:0)
11936:3] ..OnStartupComplete(custom[]:0)
11936:3] ..OneNote backup folder:: C:\Users\Cristi\AppData\Local\Microsoft\OneNote\15.0\Backup
11936:3] ..OneNote default folder: C:\Users\Cristi\Documents\OneNote Notebooks
11936:3] ..OneNote unfiled folder: C:\Users\Cristi\Documents\OneNote Notebooks\My Notebook\Quick Notes.one

**Comment by stevencohn** [comment]
Do you know which version of .NET you have installed?

**Comment by 1001QAdotNET** [comment]
I have these
![image](https://user-images.githubusercontent.com/17384308/102233524-5ffc2300-3ebe-11eb-9db7-d4051875ba4c.png)

**Comment by 1001QAdotNET** [comment]
Even with the new debuging release I am still having the same issue
Removed 3.9.3
installed the debugging release but no luck
then disabled
then enabled 
still no luck

18808:1] Starting dllhost 18808, en-US/en-US, v3.10.4, OneNote 15.0, Office 15.0
18808:1] ..OnConnection(ConnectionMode:ext_cm_Startup,1)
18808:1] ..OneAddInsUpdate(custom[]:0)
18808:1] ..building ribbon
18808:1] ..building ribbon colorizer commands
18808:1] ..loading System.Runtime.CompilerServices.Unsafe
18808:1] ..loaded System.Runtime.CompilerServices.Unsafe
18808:1] ..building ribbon groups
18808:1] ..building context menu
18808:1] ..building context menu search engines
18808:1] ..OnStartupComplete(custom[]:0)
18808:1] ..OneNote backup folder:: C:\Users\Cristi\AppData\Local\Microsoft\OneNote\15.0\Backup
18808:1] ..OneNote default folder: C:\Users\Cristi\Documents\OneNote Notebooks
18808:1] ..OneNote unfiled folder: C:\Users\Cristi\Documents\OneNote Notebooks\My Notebook\Quick Notes.one
18808:1] ..defining hotkeys
18808:1] ..ready
18808:1] OneAddInsUpdate(custom[]:0)
18808:1] OnDisconnection(RemoveMode:ext_dm_UserClosed,0)
18808:1] closing log

18588:1] Starting dllhost 18588, en-US/en-US, v3.10.4, OneNote 15.0, Office 15.0
18588:1] ..OnConnection(ConnectionMode:ext_cm_AfterStartup,1)
18588:1] ..OneAddInsUpdate(custom[]:0)
18588:1] ..building ribbon
18588:1] ..building ribbon colorizer commands
18588:1] ..loading System.Runtime.CompilerServices.Unsafe
18588:1] ..loaded System.Runtime.CompilerServices.Unsafe
18588:1] ..building ribbon groups
18588:1] ..building context menu
18588:1] ..building context menu search engines

**Comment by 1001QAdotNET** [comment]
To make sure nothing is hiding out in plain sight I tried to use some of the shortcuts https://github.com/stevencohn/OneMore/wiki#keys bit none worked so I guess it is dead not hidden

**Comment by stevencohn** [comment]
I'm not surprised. Seems like it's not loading but I don't know where it's stopping. Can you use the latest 3.10.4 and send me the log please?

**Comment by 1001QAdotNET** [comment]
the above are with 3.10.4

**Comment by stevencohn** [comment]
Here’s an idea... do any OneNote commands show up int the ribbon customize dialog?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by 1001QAdotNET** [comment]
You mean OneMore right?
No they don't

**Comment by 1001QAdotNET** [comment]
What is the diff between the two 3.9.3 that works fine and 3.10.X

**Comment by stevencohn** [comment]
Mainly I included a nuget package. It's crazy that I have four machines that work fine. I'll have to try to pull out that package and try a different approach. That's the only thing I can think of. 

That's the line in the log about CompilerServices.unsafe. That's a dependency of System.Text.Json

**Comment by stevencohn** [comment]
I really appreciate your patience and am hoping I can ask you to try another installer. It's the same release but "b" versions of the installers here https://github.com/stevencohn/OneMore/releases/tag/3.10.4

**Comment by 1001QAdotNET** [comment]
Same behavious
Try to generate two versions of the same release with new (you just did) and the old installer so we can make sure that is the smoking gun

**Comment by stevencohn** [comment]
I added a "c" version to https://github.com/stevencohn/OneMore/releases/tag/3.10.4

This completely removes any reference to the System.Text.Json nuget package.

**Comment by 1001QAdotNET** [comment]
nope ...I removed Onetastic (which I did not use anyway) but that did not solve the problem
I also used the Windows option to repair onenote -no luck with any of the above
Is this release using the old installer or you are still trying to adjust the new one?

**Comment by 1001QAdotNET** [comment]
BTW I also have OneNote 2010 on my system -that one does not see the addon either
Both are seeing onetastic which I reinstalled

**Comment by stevencohn** [comment]
There is no "new" or "old" installer. I haven't changed the installer. The only difference should have been adding that nuget package. I'll double check though... but I'm concerned that the only way I'll figure this out is to see it happen and I haven't been able to reproduce it.

**Comment by stevencohn** [comment]
Oh my. I just noticed some left out references in the installer.... I'll get rid of them

**Comment by stevencohn** [comment]
"d" installer is ready. 

Apparently, Visual Studio isn't great at removing references from the installer even though you delete them from the VS editor. It still doesn't make sense in my mind why these extra assemblies would break it though so I'm keeping fingers crossed.

**Comment by 1001QAdotNET** [comment]
same problem..

**Comment by 1001QAdotNET** [comment]
it installs fine on my wife's computer also running the same version

**Comment by 1001QAdotNET** [comment]
Would it be possible to register the add on manually ? If yes what file ?
I tried to re-register RiverOnemore AddIn.dll but it is telling me it is not a valid addon
When I check the installed one it says location mscoree.dll Is that normal?

**Comment by stevencohn** [comment]
Wait. Really? So it works OK on your wife's computer? You can see it in the ribbon bar and it works? Oh, if you can do some comparison that would be awesome. I would look at Windows updates, .NET versions, and then look at the Registry entries (see the COM Registration section on this page: https://github.com/stevencohn/OneMore/wiki/Developer-Notes)

Also when you uninstall it, make sure the ProgramFilesx86\River directory is entirely deleted before installing again.

Yes it registers itself as mscoree.dll. It's based on the example OneNoteVanillaAddin project that Microsoft provides. If you were trying to use regasm.exe that doesn't write all the necessary Registry entries.

**Comment by 1001QAdotNET** [comment]
what registry keys am I looking for
After deleting the following werestill there
Computer\HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\14.0\Common\CustomUIValidationCache

Computer\HKEY_CLASSES_ROOT\Installer\Assemblies\C:|Program Files (x86)|River|OneMoreAddIn|System.Net.Http.dll

I deleted any key with name or value containing onemore and reinstalled 
no luck

What keys am I looking to compare with my wife's machine

**Comment by stevencohn** [comment]
They're all list above but here they are:

HKEY_CLASSES_ROOT\AppID{88AB88AB-CDFB-4C68-9C3A-F10B75A5BC61}
HKEY_CLASSES_ROOT\WOW6432Node\CLSID{88AB88AB-CDFB-4C68-9C3A-F10B75A5BC61}
HKEY_CLASSES_ROOT\River.OneMoreAddIn
HKEY_CLASSES_ROOT\River.OneMoreAddIn.1
HKEY_CURRENT_USER\SOFTWARE\Classes\AppID{88AB88AB-CDFB-4C68-9C3A-F10B75A5BC61}
HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\OneNote\AddIns\River.OneMoreAddIn
HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\River.OneMoreAddIn.dll

**Comment by stevencohn** [comment]
I believe it is being registered. Otherwise you wouldn't see _anything_ in the log file.

**Comment by 1001QAdotNET** [comment]
is there any other way to engage it without using the tool bar ?
should I see the ddl loaded in memorory?

**Comment by stevencohn** [comment]
You'll see a dllhost.exe process that looks like this, note the {88AB... part is the same GUID it is registered under

![image](https://user-images.githubusercontent.com/9964757/102362769-c480b600-3f82-11eb-838f-31d52b82fa7e.png)

**Comment by 1001QAdotNET** [comment]
yes it is there
It think it is all about GUI or something , probably that is why it is not showing in the Customize ribbon  either

**Comment by 1001QAdotNET** [comment]
It is definitelly a GUI issue, I just tried again some of the shortcuts like F5 and Ctrl-H -the worked!

**Comment by stevencohn** [comment]
OK, the difference then is the addition of the Colorize menu. OneNote is supposed to complain if I feed it invalid XML for the ribbon bar but there's something strange that it doesn't like.

Do you have any other add-ons that it might be conflicting with?

**Comment by 1001QAdotNET** [comment]
Just onetastic which I removed and put it back
I also had the tagging one which I also removed

**Comment by stevencohn** [comment]
the other thing you could try (which shouldn't make a difference but...) is to temporarily rename your OneMore settings folder in case there's something in there it doesn't like:

%appdata%\OneMore

**Comment by 1001QAdotNET** [comment]
BINGO!

**Comment by 1001QAdotNET** [comment]
Damn what a ride, glad we sorted this out I was losing hope

**Comment by stevencohn** [comment]
And thank you! I am very grateful for your help and persistence. I analyze the code and try to figure out what it may not have liked and correct it asap. Cheers!

**Comment by stevencohn** [comment]
If you haven't wiped it out of existence yet, would you mind zipping up all the files in that folder and sending them to me?

**Comment by 1001QAdotNET** [comment]
I will send you the settings file, that is the only one that bothers it
I put back all the files one by one
Only settings.xml caused the trouble
I tried deleting the settings one by one but I can't figure out what is the problem

`
<?xml version="1.0" encoding="utf-8"?>
<settings>
  <images width="500" />
  <runPlugin>
    <cmd>C:\Utils\XMLQuireWin8\Application Files\XMLQuire_1_1_7_804\XMLQuire.exe</cmd>
    <args></args>
    <update>false</update>
    <child>true</child>
  </runPlugin>
  <outline>
    <addNumbering>false</addNumbering>
    <alphaNumbering>false</alphaNumbering>
    <numericNumbering>false</numericNumbering>
    <cleanupNumbering>false</cleanupNumbering>
    <indent>false</indent>
    <indentTagged>true</indentTagged>
    <removeTags>true</removeTags>
    <tagSymbol>15</tagSymbol>
  </outline>
  <ContextMenuSheet>
    <ribPasteRtfButton>true</ribPasteRtfButton>
    <ribEditMenu>true</ribEditMenu>
    <ribHighlightButton>true</ribHighlightButton>
    <ribCleanMenu>true</ribCleanMenu>
    <ribRemoveEmptyButton>true</ribRemoveEmptyButton>
    <ribRemoveSpacingButton>true</ribRemoveSpacingButton>
    <ribTrimButton>true</ribTrimButton>
    <ribSnippetsMenu>true</ribSnippetsMenu>
    <ribColorizeMenu>true</ribColorizeMenu>
  </ContextMenuSheet>
  <HighlightsSheet>
    <theme>Faded</theme>
  </HighlightsSheet>
  <RibbonBarSheet>
    <editCommands>true</editCommands>
    <editIconsOnly>true</editIconsOnly>
    <formulaCommands>true</formulaCommands>
    <formulaIconsOnly>true</formulaIconsOnly>
  </RibbonBarSheet>
  <SearchEngineSheet>
    <engines></engines>
    <engines>
      <engine>
        <image>iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEE0AABBNAWeMAeAAAAIBSURBVDhPjVLPS1RRFJ5NUNOq/yNkiESZ2qhgZTsDfQW2UEjENgW6aRdUtDAodGjQ2WhpPSUj/EVoIboJKjVGnXQlQuibH4wwTr43793Pc857T2dykD443Hfud77v3PPuDfwLpdQlx3Gitm1vUlhebNDeawAhr+wkiAxSUYzWU8Hm1OScJ3NB+0HqssAFyjSRH3uLTEcLkg1XYVwLI3PvDvbfDwrHoNr5EhO/s2PsINOmwairLBvZ7vtiwOCRRMwz8wa7p1ubpDB54wpyA32wfi3Biq8gF4sgfbcR9vaWiIsQ4u5R/iosvUKqsQpGfTWJloUtgeN4H8cgbSRA82xyYn+rhPkpiPybTiH/B6RNsIHFSeHLeRRmz0ClpoT0UfskdyIe6X+FI615bDAXdA2Sk0L6KGfQNVxqsMHJ3uJFpGcv4ONKj5Dl8PjDgRj0TB5ITtp1/on8wvBj7Tmu6zdRrTfhp7EqBcVI/HFQ/8w9wULClj3S9vIjCnFi0iTN0w9w+d0thHUNL5cH8X03TmZr6I/raBiKouZpFu2xPJRiBV29UhXyFvyr3N1PQfNMyoU2/gI7WVdNmj4RM/hZ0jzzTJiOhZHfE2j53I3wqCanuT3zELHVMeQL7uwk/krLWU/ugk38k5wG7kxLqbgYRIaoKEInSvA1ebFOe71HMx8hEDgE+OD87ZIzGLwAAAAASUVORK5CYII=</image>
        <name>Google</name>
        <uri>https://www.google.com/search?q={0}</uri>
      </engine>
      <engine>
        <image>iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEE0AABBNAWeMAeAAAAIBSURBVDhPjVLPS1RRFJ5NUNOq/yNkiESZ2qhgZTsDfQW2UEjENgW6aRdUtDAodGjQ2WhpPSUj/EVoIboJKjVGnXQlQuibH4wwTr43793Pc857T2dykD443Hfud77v3PPuDfwLpdQlx3Gitm1vUlhebNDeawAhr+wkiAxSUYzWU8Hm1OScJ3NB+0HqssAFyjSRH3uLTEcLkg1XYVwLI3PvDvbfDwrHoNr5EhO/s2PsINOmwairLBvZ7vtiwOCRRMwz8wa7p1ubpDB54wpyA32wfi3Biq8gF4sgfbcR9vaWiIsQ4u5R/iosvUKqsQpGfTWJloUtgeN4H8cgbSRA82xyYn+rhPkpiPybTiH/B6RNsIHFSeHLeRRmz0ClpoT0UfskdyIe6X+FI615bDAXdA2Sk0L6KGfQNVxqsMHJ3uJFpGcv4ONKj5Dl8PjDgRj0TB5ITtp1/on8wvBj7Tmu6zdRrTfhp7EqBcVI/HFQ/8w9wULClj3S9vIjCnFi0iTN0w9w+d0thHUNL5cH8X03TmZr6I/raBiKouZpFu2xPJRiBV29UhXyFvyr3N1PQfNMyoU2/gI7WVdNmj4RM/hZ0jzzTJiOhZHfE2j53I3wqCanuT3zELHVMeQL7uwk/krLWU/ugk38k5wG7kxLqbgYRIaoKEInSvA1ebFOe71HMx8hEDgE+OD87ZIzGLwAAAAASUVORK5CYII=</image>
        <name>Google Books</name>
        <uri>https://www.google.com/search?q={0}</uri>
      </engine>
      <engine>
        <image>iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAOwwAADsMBx2+oZAAAAopJREFUOE+Nkt1LU3Ecxnsxoouiq+hfiOgi8LYSugrpohZkODGDsEDMDCICS85wDG2ZeaH4Nq1t7U3tpA5z6dl8mbk557FNx9ymnk3n9GyzrVZdtKdzxskaqfSBB87vd57nOd8vnH08FEXlxJZHxUGKECf9lDidTh/OvPhfvrOLFpZWYZWSID77Gkm/ifkatN0DsF+w7E0yMOxdNxOImCWCavBlxYp0mj0mWHYn4R0URZ2vfvJB/utxbpKIWYoY9/xtw0uzS+wpXpub6aNCJJtUyFEQ+6RDmCIQc7QhNtOBiKUGW24dXMN2DLbbYVI4sDizauFWOijE/sBdHkgEzIvhEQnWhwmETQQYYzVGNWo0PTRCVqjBs2I9aCrAWXGIzziX4sfbjA7nKL3cnSlJMVPno65urI4Q3Ao6dD6vwp3L5SAK1JDd+F3g3y5oHZh2iqq16DDOkJkCns/ByUtxn7mfHrJDWtqCynw5pNdUkF1/s10w5YqcbO6zOwskOuTdV2QXUBRyrG/nx5or+lFbpENdkR51hVrUcpKXGDDLFbQap5evVKlxoaIju4AbLcdOzhtauDA/cu/LicS4wVU/Qbq7yMZJ9LXZ0Kz7CNETDc6Vt3MFiuwChmGO2Po90YbbPai/pYdZ42T4e/4PXfGzaoXWBnG1HvmPlLj4oDMT5qfIWsE+5DszZnCVWbRzZQF6rTjIJq76QtHGFwbrfIm0B8r3NKwuBuUNA8jjwnyB0kQHhHg24WTyhMHiDlU2DeKmrBeqDzSSqR/QjMyhSNoN0VNtRqXydxAi/0KOuU+T4567Dm8YgbUtbGyl4GGi4M9/S7DvDDnpiz/uGsdeEqw7Y/OGznYNLeTuroXcX3QiDj+j2eaOAAAAAElFTkSuQmCC</image>
        <name>Google Scholar</name>
        <uri>https://scholar.google.com/scholar?q={0}</uri>
      </engine>
      <engine>
        <image>iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEE0AABBNAWeMAeAAAADsSURBVDhPpZMBDYQwDEVBBRIQgQKkIGEG5gALGMAGBjCAjO3yxv2lg+XChSU/t3bt7++/0OiEEOI/+Ladp1bwBK+ahfcE27bFYRiQE+d5TslxHHN8HEd6B9yXZYld16Wafd9PBTy0bRvXdU0ExH3fF4TkuAPIdM8rUGQfNAlS732RFzHIBKyCbH6VQwVQDBhk48JEHm3BNE1JhWLInXM5BgUBcvECc4ghgwDZxBBaL0BBAJDMFMjYlf3J0QjBtf5GwDRUaBX9Qxhs/RFuBADZ1mkmX80TqgTywOK6u8DnUCV5gtSsUyv4hbOraT5zNVPvKCo8XgAAAABJRU5ErkJggg==</image>
        <name>Wikipedia</name>
        <uri>https://en.wikipedia.org/wiki/{0}</uri>
      </engine>
    </engines>
  </SearchEngineSheet>
</settings>
`

**Comment by 1001QAdotNET** [comment]
OK it is this one

`<ContextMenuSheet>
    <ribPasteRtfButton>true</ribPasteRtfButton>
    <ribEditMenu>true</ribEditMenu>
    <ribHighlightButton>true</ribHighlightButton>
    <ribCleanMenu>true</ribCleanMenu>
    <ribRemoveEmptyButton>true</ribRemoveEmptyButton>
    <ribRemoveSpacingButton>true</ribRemoveSpacingButton>
    <ribTrimButton>true</ribTrimButton>
    <ribSnippetsMenu>true</ribSnippetsMenu>
    <ribColorizeMenu>true</ribColorizeMenu>
  </ContextMenuSheet>`

**Comment by stevencohn** [comment]
It's the ribColorizeMenu item. I have the same one in mine but it works OK. 

But this is good because I now have something to debug. Thanks again!

**Comment by 1001QAdotNET** [comment]
this entry
    <ribEditMenu>true</ribEditMenu>

**Comment by stevencohn** [comment]
Yes! It's the combination of both the Edit menu (which includes the Colorize menu) and adding the Colorize menu itself - both as context menus. Now you're getting duplicate IDs in the XML which is what OneNote doesn't like.

Whew! I feel so much better now. I'll get right on it.

**Comment by 1001QAdotNET** [comment]
Perhaps it is worth writing a Troubleshooting section with the above (things to check) we did look at a few things that might be useful and save you time if they are on an wiki page

**Comment by stevencohn** [comment]
Good idea. I started one. I'll add to it.... https://github.com/stevencohn/OneMore/wiki/Troubleshooting

**Comment by darbin-reyes** [comment]
I tried all the suggestions here and none worked for me.  I was able to fix the issue reported by @dkter with the following steps.

On Windows 10 OS Build 21390.2025:

1. Uninstall OneMore add-in.
2.  Go to Settings-> Apps & Features 
3. Search for "365". Click "Modify".
![image](https://user-images.githubusercontent.com/86790923/127547055-92171771-99de-43b6-923e-87b1ca9ce329.png)
4. Select "Quick Repair". Click "Repair". You will need to close any MS365 apps.
![image](https://user-images.githubusercontent.com/86790923/127547115-1b494452-b3d9-424e-9adc-c11d8556b6d1.png)
5. When the repair completes, get the latest version of OneMore and install it (v3.26 in my case). 
6. Once the OneMore installation completes, open OneNote and the load error should not occur.
